### DISCUSSION TITLE
can I use videos/gif as wallpaper?

### DISCUSSION THREAD

**Discussion by wowxiz** [Godiesc/firefox-gx#148] [Q&A] [answered]
I know how to do fornighttab, but I would like to use it on the normal firefox home page. 
if I change the ogx_root-personal.css lines 19 and 24 for a file with .gif or .mp4 it will break anything? or I should link it to my uploaded video?

**Comment by Godiesc** [comment, marked as answer]
hi, I tested with some formats of images and .gif Firefox admite them, you just try it and see if work with .mp4, I don't know. And The theme just display the image, I don't think issues appear, you can use local file or the link, there is no problem.
